docs/migrating.md

```diff
@@ -0,0 +1,258 @@
+# Migrating from `react-stripe-elements`
+
+This guide includes of step by step instructions for migrating your Stripe
+integration from
+[`react-stripe-elements`](https://github.com/stripe/react-stripe-elements) to
+`@stripe/react-stripe`.
+
+- Prefer something a little more comprehensive? Check out the official
+  [React Stripe docs](https://stripe.com/docs/stripe-js/react-stripe).
+- Prefer something a little more light weight? Check out our
+  [existing examples](https://github.com/stripe/react-stripe/tree/master/examples).
+
+---
+
+## Prerequisites
+
+`@stripe/react-stripe` depends on the React Hooks API. The minimum supported
```

Comment:
I think the "If you are using hooks currently, then you should be fine. " can be removed (has a weird casual tone and seems redundant).